science_requirements.txt

```diff
@@ -0,0 +1,117 @@
+alabaster==0.7.12
+appdirs==1.4.4
+astroid==2.4.2
+async-generator==1.10
+atomicwrites==1.4.0
+attrs==20.3.0
+autopep8==1.5.4
+Babel==2.9.0
+backcall==0.2.0
+bcrypt==3.2.0
+black==20.8b1
+bleach==3.2.1
+cffi==1.14.3
+chardet==3.0.4
+click==7.1.2
+cloudpickle==1.6.0
+colorama==0.4.4
+cryptography==3.2.1
+decorator==4.4.2
+defusedxml==0.6.0
+diff-match-patch==20200713
+docutils==0.16
+entrypoints==0.3
+flake8==3.8.4
+helpdev==0.7.1
+idna==2.10
+imagesize==1.2.0
+importlib-metadata==2.0.0
+intervaltree==3.1.0
+ipykernel==5.3.4
+ipython==7.19.0
+ipython-genutils==0.2.0
+isort==5.6.4
+jedi==0.17.2
+Jinja2==2.11.2
+jsonschema==3.2.0
+jupyter-client==6.1.7
+jupyter-core==4.7.0
+jupyterlab-pygments==0.1.2
+keyring==21.5.0
+lazy-object-proxy==1.4.3
+MarkupSafe==1.1.1
+mccabe==0.6.1
+mistune==0.8.4
+mypy-extensions==0.4.3
+nbclient==0.5.1
+nbconvert==6.0.7
+nbformat==5.0.8
+nest-asyncio==1.4.3
+numpydoc==1.1.0
+packaging==20.4
+pandocfilters==1.4.3
+paramiko==2.7.2
+parso==0.7.0
+pathspec==0.8.1
+pathtools==0.1.2
+pexpect==4.8.0
+pickleshare==0.7.5
+pluggy==0.13.1
+prompt-toolkit==3.0.8
+psutil==5.7.3
+ptyprocess==0.6.0
+pycodestyle==2.6.0
+pycparser==2.20
+pydocstyle==5.1.1
+pyflakes==2.2.0
+Pygments==2.7.2
+pylint==2.6.0
+pyls-black==0.4.6
+pyls-spyder==0.1.1
+PyNaCl==1.4.0
+pyparsing==2.4.7
+PyQt5==5.12.3
+PyQt5-sip==12.8.1
+PyQtWebEngine==5.12.1
+pyrsistent==0.17.3
+python-dateutil==2.8.1
+python-jsonrpc-server==0.4.0
+python-language-server==0.36.1
+pytz==2020.4
+pywin32==300; sys_platform == 'win32'
+pywin32-ctypes==0.2.0; sys_platform == 'win32'
+pyzmq==20.0.0
+QDarkStyle==2.8.1
+QtAwesome==1.0.1
+qtconsole==4.7.7
+QtPy==1.9.0
+regex==2020.11.13
+requests==2.25.0
+rope==0.18.0
+six==1.15.0
+snowballstemmer==2.0.0
+sortedcontainers==2.3.0
+Sphinx==3.3.1
```

Comment:
spyder depends on shpinx and jupyter? wow))

Reply:
Yes this is literally the output of `pip freeze`